src/darwin/Framework/Matter.xcodeproj/project.pbxproj

```diff
@@ -2562,6 +2579,7 @@
 		BA09EB792474882200605257 /* Release */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
```

Comment:
?

Reply:
Also no clue, I'm curious what changed this